utils/post.js

```diff
@@ -163,15 +164,16 @@ const modifyAnonymousAndBlockPost = async (
   return feedWithAnonymous;
 };
 
-const modifyAnonimityPost = async (item, isBlurredPost = true) => {
+const modifyAnonimityPost = async (item, isBlurredPost = true, anonymUserId) => {
   const newItem = {...item};
 
   if (newItem.anonimity) {
+    const isOwnAnonPost = newItem?.actor?.id === anonymUserId;
+    newItem.isBlurredPost = isOwnAnonPost ? false : isBlurredPost;
     newItem.actor = {};
     newItem.to = [];
     newItem.origin = null;
     newItem.object = '';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the function `modifyAnonimityPost` has a new parameter `anonymUserId` added at the end, after a parameter with a default value (`isBlurredPost = true`). This could lead to confusion and potential errors. Parameters with default values should be placed at the end of the parameter list.
```diff
- const modifyAnonimityPost = async (item, isBlurredPost = true, anonymUserId) => {
+ const modifyAnonimityPost = async (item, anonymUserId, isBlurredPost = true) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->